_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: Fixed crashers and dropped KubeVirt support
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 178.
+
+### KubeVirt support has been removed
+
+Cockpit shipped `cockpit-kubernetes` for managing Kubernetes and OpenShift deployments in previous releases. Since its introduction, important points have come to light:
+
+1. The OpenShift Console is adding support for [KubeVirt](http://kubevirt.io/) by [merging in Tetontic and Quay](https://www.redhat.com/en/about/press-releases/red-hat-unveils-roadmap-coreos-integration-red-hat-openshift). It will become _the_ place for managing KubeVirt-based VMs that run in OpenShift.
+
```

Comment:
That's a bit confusing. What merges is Tectonic/Quay (I don't know which part is the UI) and the OpenShift Console. I'd really just leave that out and say "The OpenShift Console is adding support for [KubeVirt](http://kubevirt.io/)."